### ISSUE TITLE
Resnet50 on Blackhole: Using pre-trained data gives bad PCC

### ISSUE THREAD

**Issue by mywoodstock** [tenstorrent/tt-metal#17558]
Using `pretrained_weight` on blackhole gives bad PCC on Resnet50 demo (~0.795), while without pretrained weights, the PCC is good (~0.998). 

This has been tracked down to transpose WH operation in the Fold op of resnet https://github.com/tenstorrent/tt-metal/issues/18522. Further, the transpose WH issue is due to lack of narrow_row support in the corresponding LLKs https://github.com/tenstorrent/tt-metal/issues/18528. 

The test with pre-trained weights is:

```
pytest "tests/ttnn/integration_tests/resnet/test_ttnn_functional_resnet50.py::test_resnet_50[pretrained_weight_true-batch_size=16-act_dtype=DataType.BFLOAT8_B-weight_dtype=DataType.BFLOAT8_B-math_fidelity=MathFidelity.LoFi-device_params={'l1_small_size': 24576}]"
```

The test is currently skipped.

**Comment by ejouretTT** [comment]
Requires further debugging. PCC – measure of accuracy.

**Comment by pavlejosipovic** [comment]
@pavlepopovic fyi

**Comment by ejouretTT** [comment]
Issue does not show up on any of the unit tests. Continuing to look through all the ops.

**Comment by mywoodstock** [comment]
I was further able to track the issue to LLK: https://github.com/tenstorrent/tt-metal/issues/18528

**Comment by ejouretTT** [comment]
@mywoodstock the issue you have linked is closed so is this resolved or should the issue referenced be reopened?

**Comment by mywoodstock** [comment]
@ejouretTT this issue is actually unrelated to the case I earlier mentioned in the description

**Comment by skrsticTT** [comment]
Closing this as #18787 is merged
